### ISSUE TITLE
meta dataset used in training

### ISSUE THREAD

**Issue by Zhen-Zohn-WANG** [deepmind/leo#15]
Thank you for this great work! 
I noticed that a PyTorch implementation [1] states that the model is trained on both the training set and validation set in the LEO paper.
I wonder to know if this TensorFlow implementation is trained on both the training set and validation set?

Thank you.

[1] https://github.com/timchen0618/pytorch-leo
